python/brainyflow.py

```
@@ -118,95 +132,101 @@
+        
+        def __rshift__(self, other: 'BaseNode') -> 'BaseNode':
             """Implement - "action" >> node_b syntax"""
             return self.node.on(self.action, other)
-
-    def get_next_nodes(self, action=DEFAULT_ACTION):
+    
+    def get_next_nodes(self, action: Action = DEFAULT_ACTION) -> List['BaseNode']:
         """Get successor nodes for a specific action."""
         next_nodes = self.successors.get(action, [])
         if not next_nodes and action != DEFAULT_ACTION and self.successors:
             warnings.warn(f"Flow ends: '{action}' not found in {list(self.successors.keys())}")
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)